### PR TITLE
Create runs component model

### PR DESCRIPTION
**Pull request by jeremie74**

### CODE REVIEW

File: libs/contract/src/lib/term/form/runs/runs-form.component.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<form [formGroup]="form">
```
Comment (GrandSchtroumpf) [inline libs/contract/src/lib/term/form/runs/runs-form.component.html:1]: As this control will never be a form on its own, better not to use a form. Best is to copy paste existing code from avails: 
```html
<article fxLayout="column" [formGroup]="form">
  <p>Broadcasts</p>
  <mat-form-field appearance="outline">
    <mat-label>Number of Broadcasts</mat-label>
    <input matInput type="number" placeholder="Number of Broadcasts" formControlName="broadcasts">
  </mat-form-field>
  <p>Catch Up Rights</p>
  <div fxFlex fxLayout fxLayoutGap="16px" fxLayoutAlign="start space-between" formGroupName="catchup">
    <mat-form-field appearance="outline" fxFlex>
      <mat-label>Start Date</mat-label>
      <input matInput [matDatepicker]="datePicker" placeholder="MM/DD/YYYY" formControlName="from">
      <mat-datepicker-toggle matPrefix [for]="datePicker">
        <mat-icon matDatepickerToggleIcon svgIcon="calendar_today"></mat-icon>
      </mat-datepicker-toggle>
      <mat-datepicker #datePicker></mat-datepicker>
    </mat-form-field>
    <mat-form-field appearance="outline" fxFlex>
      <mat-label>Duration</mat-label>
      <input matInput type="number" placeholder="Duration" formControlName="duration">
    </mat-form-field>
    <mat-form-field appearance="outline" fxFlex>
      <mat-label>Period</mat-label>
      <mat-select matInput placeholder="Period" formControlName="period">
        <mat-option *ngFor="let period of periods" [value]="period">
          {{ period | titlecase }}
        </mat-option>
      </mat-select>
    </mat-form-field>
  </div>
</article>
```

File: libs/contract/src/lib/term/form/runs/runs-form.component.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.catchup mat-form-field {
+    padding-right: 12px;
```
Comment (GrandSchtroumpf) [inline libs/contract/src/lib/term/form/runs/runs-form.component.scss:2]: You should not override padding of an existing component. Especially the mat-form-field. You can only apply margin to organise them between each other if needed.

**Review by GrandSchtroumpf — CHANGES_REQUESTED** [review]
You should also reuse the `runs-form` in the avails component to make sure there is no duplication of code: 
`apps\catalog\catalog\src\app\marketplace\title\avails\avails.component.html`

File: apps/catalog/catalog/src/app/marketplace/title/avails/avails.component.html [hunk 3]
```diff
@@ -100,34 +100,7 @@ <h5>Request Specific Versions</h5>
               </mat-panel-title>
             </mat-expansion-panel-header>
             <article class="expansion-panel-content" fxLayout="column" fxFlex formGroupName="runs">
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/marketplace/title/avails/avails.component.html:102]: You should remove this article now.

**Review by GrandSchtroumpf — APPROVED** [review]

**Comment by jeremie74** [comment]
> You should also reuse the `runs-form` in the avails component to make sure there is no duplication of code:
> `apps\catalog\catalog\src\app\marketplace\title\avails\avails.component.html`

Done

**Pull request merged**